.cirrus.yml

```diff
@@ -646,6 +736,30 @@ local_system_test_task: &local_system_test_task
     always: *logs_artifacts
 
 
+# Always run subsequent to integration tests.  While parallelism is lost
+# with runtime, debugging system-test failures can be more challenging
+# for some golang developers.  Otherwise the following tasks run across
+# the same matrix as the integration-tests (above).
```

Comment:
Minor nit: The last part of this comment is wrong, you're not adding the full compliment of system-test flavors (which is fine).  No need to re-run all the tests just for this change, but if you do have to re-push, I'd suggest maybe just nuke all/most of this comment block.

Reply:
done.